src/index.ts

```diff
@@ -622,5 +621,76 @@ function newEventDispatcher(sdkKey: string): EventDispatcher {
     { isOffline: () => false, onNetworkStatusChange: () => {} };
   const networkStatusListener =
     typeof window !== 'undefined' ? new BrowserNetworkStatusListener() : emptyNetworkStatusListener;
-  return newDefaultEventDispatcher(eventQueue, networkStatusListener, sdkKey);
+  return newDefaultEventDispatcher(
+    eventQueue as NamedEventQueue<Event>,
```

Comment:
You could also explicitly type it on line 616 above, that's slightly cleaner IMHO